src/main/java/de/zalando/ep/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -611,9 +611,6 @@ void processContainerAction(final DockerSeleniumContainerAction action, final St
             if (DockerSeleniumContainerAction.STOP_RECORDING == action) {
                 copyVideos(containerId);
             }
-            if (DockerSeleniumContainerAction.TRANSFER_LOGS == action) {
```

Comment:
@yosserO I know that the `transfer-logs.sh` does not exist in the `elgalu/selenium` image, and it is not correct to invoke the action a few lines [above](https://github.com/zalando/zalenium/pull/1053/files#diff-cdb17f721a30ede405aeccaa422c3006L608), but removing this `if` basically means that no logs will be copied when someone is running Zalenium in k8s or docker in a single host. So deleting this would break it for those environments.

Perhaps a fix to avoid executing that command unnecessarily would be to avoid executing lines [607-608](https://github.com/zalando/zalenium/pull/1053/files#diff-cdb17f721a30ede405aeccaa422c3006L607) when the action is `TRANSFER_LOGS`?

Reply:
@diemol , thank you for you feedback. My Bad I didn't consider the other environments when doing the changes. I will revert them and wrap the lines 607-608 with an if condition to avoid executing the script when the action is TRANSFER_LOGS.